### PR TITLE
Fix broken links of Callback

### PR DESCRIPTION
**Pull request by gadagashwini**

### CODE REVIEW

**Review by fchollet — APPROVED** [review]

**Comment by fchollet** [comment]
> keras/engine/training.py:21[6](https://github.com/keras-team/keras/runs/7916403603?check_suite_focus=true#step:6:7)3:81: E501 line too long (95 > 80 characters)

Please fix

**Comment by gadagashwini** [comment]
Fixed!

File: keras/engine/training.py [hunk 1]
```diff
@@ -2160,7 +2160,8 @@ def predict(
                 run until the input dataset is exhausted.
             callbacks: List of `keras.callbacks.Callback` instances.
                 List of callbacks to apply during prediction.
-                See [callbacks](/api_docs/python/tf/keras/callbacks).
+                See [callbacks](https://www.tensorflow.org/api_docs/python/tf/
+                keras/callbacks).
```
Comment (fchollet) [inline keras/engine/training.py:2164]: Please make sure the link fits on a single line

Comment (gadagashwini) [reply keras/engine/training.py]: Made the changes

File: keras/engine/training.py [hunk 2]
```diff
@@ -2160,7 +2160,8 @@ def predict(
                 run until the input dataset is exhausted.
             callbacks: List of `keras.callbacks.Callback` instances.
                 List of callbacks to apply during prediction.
-                See [callbacks](/api_docs/python/tf/keras/callbacks).
+                See [callbacks]
+                (https://www.tensorflow.org/api_docs/python/tf/keras/callbacks).
```
Comment (fchollet) [inline keras/engine/training.py:2164]: The `(` should be next to the `]` in order for the markdown to render properly.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the update!

**Comment by fchollet** [comment]
The issue appears to be fixed at head. Thanks!

**Pull request closed without merging**